src/client/src/ui/workspace/selectors.ts

```diff
@@ -1,7 +1,7 @@
 import { createSelector } from 'reselect'
 import { GlobalState } from 'StoreTypes'
 
-const makePortalProps = key => ({
+const makePortalProps = (key: string) => ({
```

Comment:
`createPortalProps`? 

`make` is reserved for non-returning side-effects … like a command line function.

Reply:
Just adding the types :)

Reply:
Could you provide evidence for that assertion @evanrs? I have not come across that practice yet.

For example, the Redux docs themselves use make: https://redux.js.org/recipes/computingderiveddata.